saleor/graphql/translations/types.py

```diff
@@ -71,6 +161,42 @@ class Meta:
         ]
 
 
+class ProductTranslatableContent(CountableDjangoObjectType):
+    translation = graphene.Field(
+        ProductTranslation,
+        language_code=graphene.Argument(
+            LanguageCodeEnum,
+            description="A language code to return the translation for.",
+            required=True,
+        ),
+        description=("Returns translated Product fields for the given language code."),
+        resolver=resolve_translation,
+    )
+    product = graphene.Field(
+        "saleor.graphql.product.types.products.Product",
+        description="Represents an individual item for sale in the storefront.",
+    )
+
+    class Meta:
+        model = product_models.Product
+        interfaces = [graphene.relay.Node]
+        only_fields = [
+            "description",
```

Comment:
I see at least 4 types that require the same `only_fields`. Maybe we could extract them to single variable?